Service/FilterService.php

```diff
@@ -70,21 +70,44 @@ public function __construct(
     /**
      * @param string $path
      * @param string $filter
+     *
+     * @return bool Returns true if the cache is busted
      */
     public function bustCache($path, $filter)
     {
-        $basePathContainer = new FilterPathContainer($path);
-        $filterPathContainers = [$basePathContainer];
-
-        if ($this->webpGenerate) {
-            $filterPathContainers[] = $basePathContainer->createWebp($this->webpOptions);
-        }
+        $busted = false;
 
-        foreach ($filterPathContainers as $filterPathContainer) {
+        foreach ($this->buildFilterPathContainers($path) as $filterPathContainer) {
             if ($this->cacheManager->isStored($filterPathContainer->getTarget(), $filter)) {
                 $this->cacheManager->remove($filterPathContainer->getTarget(), $filter);
+
+                $busted = true;
             }
         }
+
+        return $busted;
+    }
+
+    /**
+     * @param bool $forced Force warm up cache
+     *
+     * @return bool Returns true if the cache is warmed up
+     */
+    public function warmsUpCache(
```

Comment:
why warmsUpCache and not warmUpCache? it feels like a request to do the warmup to me.